components/platform_crypto/shave_trusted_boot/firmware.c

```diff
@@ -94,6 +339,11 @@ int reset(void *start_address,
       &&
       (memcmp(last_measure,expected_measure_,MEASURE_SIZE) != 0))
     return HASH_MISMATCH;
+#else
+  if (!xmss_verify_signature(&public_key, start_address, region_size, &xmss_signature)) {
```

Comment:
It doesn't look like `public_key` and `xmss_signature` are initialized anywhere, so won't this basically always return false?